app/containers/Pdf/PdfFuelFieldsTemplate.tsx

```diff
@@ -16,6 +16,13 @@ export const PdfFuelFieldsTemplate: React.FunctionComponent<Props> = ({
       <Text style={{fontSize: 15, letterSpacing: 2}}>
         {'\n'}Fuel:{'\n'}
       </Text>
+      {formData.comments ? (
+        <Text style={{marginBottom: 2, marginTop: 2}}>
+          {`comments: ${formData.comments}`}
+          {'\n'}
+        </Text>
+      ) : null}
+
```

Comment:
Is this needed? `PdfFieldTemplate` seems to be handling the conditional rendering

Reply:
The issue here is that by using a Custom Field we shortcut all the templating/widget customization that the jsonschema-form library gives us, and the contract is that we are responsible for 100% of the rendering.
So this object won't pass through the template we defined in `PdfFieldTemplate.tsx` from what I'm seeing.
Since this is reused between `PdfFuelFieldsTemplate.tsx` and `PdfProductionFieldsTemplate.tsx` I'd suggest making a custom component to display the comments in this case